docs/en/student-examples.md

```diff
@@ -0,0 +1,134 @@
+<div style="display: flex; align-items: center;">
+    <img src="../assets/en/join-course.png" alt="course enroll button" style="width: auto; height: auto; margin-right: 10px;">
+    <img src="../assets/en/leave-course.png" alt="course leave button" style="width: auto; height: auto;">
+</div>
+
+**Enroll**
+
+- Click on courses in the navigation bar.
+- Find the correct course using the filters on the left side of the page.
+- Click on the enroll button next to the correct course.
+- You are successfully enrolled in this course.
+
+**leave**
+- Option 1:
+  - Click on courses in the navigation bar.
+  - Find the correct course using the filters on the left side of the page.
+  - Click on the unenroll button next to the correct course.
+  - You are successfully leave from this course.
+- Option 2:
+  - Go to the detail page.
+  - Click on the unenroll button at the bottom.
+  - A popup appears.
+  - Confirm by clicking the leave button or cancel with the cancel button.
+
+## Course Details
+- Click on Dashboard in the navigation bar.
+- Scroll down to the "My Courses" section.
+- Click on the details button next to the chosen course.
+
+![course card](../assets/course-card.png)
+
+::: tip
+- If you want to view a course from a previous academic year, select the correct year in the academic year selector.
+:::
+
+## View Projects
+
+![projects list](../assets/en/project-list.png)
+
+- Option 1:
+  - Click on Dashboard in the navigation bar.
+  - At the top of the page is a section "Current Projects".
+  ::: info
+  - Here are links to all projects with a deadline within 7 days on the left and all projects on the right.
+  - With the toggle button at the top, you can view projects with an expired deadline.
+  :::
+- Option 2:
+  - Click on "Calendar" in the navigation bar.
+  - On the left, you see a calendar with the number of projects you have each day in a red circle if there are any.
+  - Select a day with projects and click on the desired project.
+- Option 3:
+  - Click on "Dashboard" in the navigation bar.
+  - Scroll down to the "My Courses" section.
+  - Click here on the course of the desired project.
+  - Under the "Current Projects" section, you will see all projects for this specific course.
+- Option 4:
+  - Click on "Projects" in the navigation bar.
+  - You will see an overview of all your projects.
+
+::: info Project card explanation
+![project card](../assets/en/project-card.png)
+- At the top of this card is the name of the course for which this project is issued. Below is the description of the course. The start date and deadline of the project are listed below. Below this is a progress bar for the entire class. On the left is how many submissions the class has already made and on the right is how many groups there are.
+Under this progress bar is the status of the tests of these submissions.
+:::
+
+## Find Teachers of a Course
+
+- Go to the detail page of the course.
+- Scroll to the section "Teachers linked to this course".
```

Comment:
Is now called "People linked to this course". It's a mix of teachers and assistants.